dizoo/evogym/config/carrier_ppo_config.py

```diff
@@ -0,0 +1,67 @@
+from easydict import EasyDict
+
+carry_ppo_config = dict(
+    exp_name='evogym_carrier_ppo_seed1',
+    env=dict(
+        env_id='Carrier-v0',
+        robot='carry_bot',
+        robot_dir='./dizoo/evogym/envs',
+        collector_env_num=4,
+        evaluator_env_num=2,
+        n_evaluator_episode=2,
+        stop_value=10,
+        manager=dict(shared_memory=False, ),
+        # The path to save the game replay
+        # replay_path='./evogym_carry_ppo_seed0/video',
+    ),
+    policy=dict(
+        cuda=True,
+        recompute_adv=True,
+        # load_path="./evogym_carry_ppo_seed0/ckpt/ckpt_best.pth.tar",
+        model=dict(
+            obs_shape=70,
+            action_shape=12,
+            action_space='continuous',
+        ),
+        action_space='continuous',
+        learn=dict(
+            epoch_per_collect=10,
+            batch_size=256,
+            learning_rate=3e-3,
+            value_weight=0.5,
+            entropy_weight=0.01,
+            clip_ratio=0.2,
+            adv_norm=True,
+            value_norm=True,
+        ),
+        collect=dict(
+            n_sample=2048,
+            unroll_len=1,
```

Comment:
simplify config fields which is the same as `default_config` and will not be modified frequently

Reply:
fixed.